index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./index.css">
+	<title>Mobile Skeleton</title>
+</head>
+<body>
+	<header id="nav-container">		
+		          <p class="welcome">welcome</p>	
+		          <nav id="mobile-nav">
+		          	 <img id="drop-btn" alt="drop menu icon" src="./images/dropMenuIcon.png">
+		          </nav>  
+    </header>
+	<section id="headline-section">
+			<h1 class="title">
+			Hey there. I’m Mariane<span id="info" class="title">I’m a software developer</span>
+		</h1>	
```

Comment:
Please add a font size to the title class and info id. As per Figma design, the title class and the info id have a font size of 40px. 
Please try to follow [the Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988) as much as you can 😊